nexus/src/external_api/http_entrypoints.rs

```diff
@@ -4194,26 +4327,52 @@ async fn image_delete(
     path_params: Path<ImagePathParam>,
```

Comment:
deprecated

Reply:
I hate these 😭.

Thanks, I'll update it.